master/test/streamdata/stream_data.go

```diff
@@ -0,0 +1,45 @@
+//go:build integration
+// +build integration
+
+package streamdata
+
+import (
+	// embed is only used in comments.
+	_ "embed"
+	"testing"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/test/migrationutils"
+)
+
+//go:embed stream_trials.sql
+var streamTrialsSQL string
+
+const latestMigration = 20230830174810
```

Comment:
question: will this require us to update on every new migration?

Reply:
I don't think so - I think this just migrates up to that point and then applies the one specified